src/OrderLinesList/OrderLinesFiltersContainer.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { get } from 'lodash';
+
+import { stripesConnect } from '@folio/stripes/core';
+import {
+  DICT_FUNDS,
+  fundsManifest,
+} from '@folio/stripes-acq-components';
+
+import {
+  MATERIAL_TYPES,
+} from '../components/Utils/resources';
+import OrderLinesFilters from './OrderLinesFilters';
+
+const OrderLinesFiltersContainer = ({ resources, activeFilters, applyFilters }) => {
+  const materialTypes = get(resources, 'materialTypes.records', []);
+  const funds = get(resources, `${DICT_FUNDS}.records`, []);
```

Comment:
Can we don't use resources and load data manually?

Reply:
yes, we can, but it's not the point of the PR. Also it's pretty improtant to understand the difference of _load data manually_ and stripes-connect `fetch: true` resources. Stripes-connect adds kind of cache layer, so we don't need to make actual calls on back-end for dictionary data. In this particular case all funds are loading, that is wrong by the meaning. In future all such places will be changed to look-up plugins, for now as temporary solution we try to use that cache thing from stripes-connect

Reply:
Thnx for clarification)